pkg/services/provisioning/provisioning.go

```diff
@@ -136,14 +136,16 @@ func (ps *provisioningServiceImpl) ProvisionDashboards() error {
 		return errutil.Wrap("Failed to create provisioner", err)
 	}
 
+	// If we fail to provision with the new provisioner, mutex will unlock and the polling we restart with the
```

Comment:
```suggestion
	// If we fail to provision with the new provisioner, the mutex will unlock and the polling will restart with the
```

Reply:
Why are you moving this comment? It looks to me as if it makes since in the old location.

Reply:
Unfortunately, I can't remember now. Looks like it's better to move it to the old place.